CHANGELOG.md

```diff
@@ -94,6 +94,53 @@
 - `merge` no longer treats `null` in the spec as a delete option but rather as a normal value
 - Combine all compression and decompression pre/postprocessors.
 
+## [0.11.12]
+
+### Fixes
+- Unix onramp: Add the missing StartStream message to handle multiple connections correctly
+
+## 0.11.10
```

Comment:
If we add the `[...]` notation it'd be great to add them to all past entries, that way it remains consistent

Reply:
Done!